performance/images.md

```diff
@@ -1,22 +1,43 @@
 # Images
 
+* [Introduction](#introduction)
+* [Picking the right image format](#picking-the-right-image-format)
+* [Image formats](#image-formats)
+  * [SVG images](#svg-images)
+  * [JPEG images](#jpeg-images)
+  * [PNG images](#png-images)
+* [Optimisation of images](#optimisation-of-images)
+* [Serving images](#serving-images)
+* [Comparing the quality of images](#comparing-the-quality-of-images)
+
 This document describes best practices for the use of images on the web at Springer Nature.
 
-In a frontend workflow we typically deal with two kinds of image:
+## Introduction
+
+In a frontend workflow we typically deal with two different categories of images:
+
+* **User interface images**. These include icons, logos, buttons, and any other image that is part of the user. Usually managed by the frontend, these are stored with other client-side resources such as templates, CSS and JS. Like those resources, they may be shared between many different pages.
+
+* **Content images**. These are unique to a page, or a handful of pages. They will usually be provided from a database or a content management system.
 
-* **User interface images**, including icons, logos, and buttons. Usually managed by the frontend, these are stored with other client-side resources such as templates, CSS and JS. Like those resources, they may be shared between many different pages.
+Most best practices described in this document apply to both categories unless otherwise indicated.
 
-* **Content images**. These are unique to a page (or a handful of pages). They will usually be provided from a database or a content management system.
+## Picking the right image format
 
-Most best practices are common to both categories.
+* Use [SVG](https://en.wikipedia.org/wiki/Scalable_Vector_Graphics) for icons, logos, and other images with sharply defined edges.
+* Use [JPEG](https://en.wikipedia.org/wiki/JPEG) for still images, photographs, and in any other cases where using a PNG would result in an image too large to be delivered over the network effectively.
+* Use [PNG](https://en.wikipedia.org/wiki/Portable_Network_Graphics) for illustrations, graphs, flat-colour graphics, images containing transparency (which JPEG doesn't support), and images containing text.
+* Use [MP4](https://en.wikipedia.org/wiki/MPEG-4_Part_14) for any kind of animated images.
+* Do not use [GIFs](https://en.wikipedia.org/wiki/GIF). For animated images, the file size of GIF files, usually an order of magnitude bigger than MP4s of similar quality, may be a barrier preventing users from accessing our content, specially in markets where slower connections and low performing devices are more common. For static images with transparency, an 8-bit PNG can be used to deliver an indexed colour image, with binary transparency, [at a smaller size than a GIF](https://helpx.adobe.com/photoshop-elements/using/optimizing-images-gif-or-png.html).
```

Comment:
What about images without transparency … PNGs still smaller?

Reply:
It should, yes. The reason why the text only mentions the transparency use case is because GIFs have been traditionally used for this purpose. But from the Adobe link mentioned:

> The PNG‑8 format uses more advanced compression schemes than GIF does, and a PNG‑8 file can be 10% to 30% smaller than a GIF file of the same image, depending on the image’s color patterns.

This is independent from the image using transparency or not.